glue/core/roi.py

```diff
@@ -463,10 +487,10 @@ class VertexROIBase(Roi):
 
     def __init__(self, vx=None, vy=None):
         """
-        :param vx: initial x vertices
-        :type vx: list
-        :param vy: initial y vertices
-        :type vy: list
+        vx : list
+             initial x vertices
```

Comment:
It looks like there are fives spaces in the indentation here but there should be four.